.gitignore

```diff
@@ -18,3 +18,6 @@ build/
 dist/
 
 .eggs/
+
+#pyenv
+.python-version
```

Comment:
I haven't seen this before: what is it?

Reply:
It's generated by pyenv: https://github.com/pyenv/pyenv . Many of us use it for virtualenv management, particularly in repos in which we develop outside of a devstack context.

Reply:
I see.  You might like to create a global gitignore file to fix this across all your repos also: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Wow, good tip, thanks for the knowledge!